webapp/channels/src/components/admin_console/system_users/system_users_filters_popover/system_users_filter_team/index.tsx

```diff
@@ -221,7 +221,7 @@ const Option = (props: OptionProps<OptionType, false>) => {
     );
 };
 
-const LoadingIndicator = () => {
+export const LoadingIndicator = () => {
```

Comment:
question: why the need to export this?

Reply:
Its being used in UserMultiSelector as well